tests/contract/test_resource_client.py

```diff
@@ -1238,7 +1248,24 @@ def test_transform_model_equal_output(resource_client):
     output_model = TRANSFORM_OUTPUT.copy()
 
     resource_client.transform_model(input_model, output_model)
-    assert input_model == TRANSFORM_OUTPUT
+    assert input_model == output_model
+
+
+def test_transform_model_equal_output_OR(resource_client):
+    input_model = {"e": "new", "a": "ValueA", "c": 1}
+    output_model = {"e": "newValue", "a": "ValueA", "c": 1}
```

Comment:
Do we have a test for the other side of the OR? It's not 100% clear of what the OR function should do, but I assume that having 
```
output_model = {"e": "newTest", "a": "ValueA", "c": 1}
```
should also make input and output be seen as equivalent

Reply:
will add parameterized output to check both the conditions